fuel-vm/src/tests/blockchain.rs

```diff
@@ -552,29 +560,37 @@ fn ldc_reason_helper(
 
 #[test]
 fn ldc_ssp_not_sp() {
-    // test ssp != sp for LDC
-    let load_contract = vec![
-        op::cfei(0x1), // sp += 1
-        op::ldc(RegId::ZERO, RegId::ZERO, RegId::ZERO), /* Load first two words from
-                        * the contract */
-    ];
+    let (load_contract, _) = script_with_data_offset!(
+        data_offset,
+        vec![
+            op::movi(0x10, data_offset as Immediate18),
+            op::cfei(0x1), // sp += 1
+            op::ldc(0x10, RegId::ZERO, RegId::ZERO), /* Load first two words from *
+                            * the contract */
```

Comment:
This loads zero words from the contract, so the comment is incorrect. I suggest loading a single byte anyway, since this test shouldn't be hitting the zero-length-load corner case.

```suggestion
            op::ldc(0x10, RegId::ZERO, RegId::ONE),
```